lms/templates/orchestrator-deployment.yaml

```diff
@@ -56,7 +56,7 @@ spec:
             - name: ORCH_SOURCE_DIALECT
               value: {{ .Values.orchestrator.sourceDialect }}
             - name: ORCH_TARGET_DIALECT
-              value: "cockroach"
+              value: {{ .Values.orchestrator.targetDialect }}
```

Comment:
You will need to define an entry for targetDialect in the values.yaml file since that doesn't exist right now.

Reply:
Done.